app/lib/common/utilities/image_upload.dart

```diff
@@ -0,0 +1,218 @@
+import 'dart:io';
+import 'dart:async';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:image_picker/image_picker.dart';
+
+class UploadImage extends StatefulWidget {
+  const UploadImage({Key key}) : super(key: key);
+  static const route = '/image_upload';
+  @override
+  _UploadImageState createState() => _UploadImageState();
+}
+
+class _UploadImageState extends State<UploadImage> {
+  PickedFile _imageFile;
+  dynamic _pickImageError;
+  File _image;
+  final ImagePicker _picker = ImagePicker();
+  final TextEditingController maxWidthController = TextEditingController();
+  final TextEditingController maxHeightController = TextEditingController();
+  final TextEditingController qualityController = TextEditingController();
+
+  void _onImageButtonPressed(ImageSource source, {BuildContext context}) async {
+    try {
+      final pickedFile = await _picker.getImage(
+        source: source,
+      );
+      setState(() {
+        _imageFile = pickedFile;
+      });
+    } catch (e) {
+      setState(() {
+        _pickImageError = e;
+      });
+    }
+  }
+
+  Widget _previewImage() {
+    //final Text retrieveError = _getRetrieveErrorWidget();
```

Comment:
- [ ] Is this meant to be there or can it be removed?

Reply:
@holtzmak Thanks Kelly. I removed it :)